pkg/istio/rbac/common/util.go

```diff
@@ -96,6 +98,24 @@ func (a *ApiHandler) Delete(item *Item) error {
 	return err
 }
 
+// CheckIfMemberIsAllUsers returns namespace for Athenz domain when role member is of form '<athenz-domain>.*'.
+// Example: domain.sub-domain.* -> domain-sub--domain
+func CheckIfMemberIsAllUsers(member *zms.RoleMember, domainName zms.DomainName) (string, error) {
+
+	if member == nil {
+		return "", fmt.Errorf("member is nil")
+	}
+
+	memberStr := string(member.MemberName)
+
+	// if member name is of the form '<athenz-domain>.*', return namespace
+	if strings.HasPrefix(memberStr, "user.") || !strings.HasSuffix(memberStr, ".*") {
```

Comment:
I think there are some cases like `cd.screwdriver.project.*` or `k8s.omega.*` could not be captured here. Should we do a strict check, equal to * here?

Reply:
Why wouldn't they get handled here? Both of those cases don't have `user.` as a prefix and have `.*` as the suffix.

Reply:
What I am trying to say is if there is non real user regex `cd.screwdriver.project.*`, the logic here will convert it to a namespace rule.

Reply:
Check with @mcieplak for athenz user naming convention, there won't be above case.

Reply:
Can you also check for prefix `unix.`? a valid user name in athenz can be `unix.<group>`,  we want to avoid creating namespace in this case.

Reply:
Isn't `cd-screwdriver-project` a K8s namespace? A namespace for all Screwdriver services?

Reply:
Not sure what those SD services would get mapped to in the K8s space though.

Reply:
For bypassing `unix.*` - https://github.com/yahoo/k8s-athenz-istio-auth/pull/58/commits/c2131ccc31aa4edd6d200d83c5067f928a93cb2c.